README.md

```diff
@@ -1 +1,42 @@
-# tekton-watcher
+# Tekton Watcher
+
+Tekton Watcher is a service to react to events that occur in Tekton Pipelines and Tasks.
+
+![Architectural diagram showing how the Kubernetes events are captured by Tekton Watcher and its relationship with GitHub](TektonWatcherDiagram.jpg)
+
+## Installation
+
+> To build Tekton Watcher it depends on [Vessel](https://github.com/nubank/vessel)
+
+Update `app/*.yaml` configurations to your needs
+
+```
+make build
+docker image load -i target/tekton-watcher.tar
```

Comment:
Just to mention, `Vessel` has a push command that pushes a tarball to a remote registry without loading it into the Docker daemon.